omnigibson/prims/prim_base.py

```diff
@@ -20,43 +21,44 @@ class BasePrim(Serializable, UniquelyNamed, Recreatable, ABC):
         unless it is a non-root articulation link.
 
     Args:
-        prim_path (str): prim path of the Prim to encapsulate or create.
+        relative_prim_path (str): prim path of the Prim to encapsulate or create.
```

Comment:
what does "relative" imply? can we make the docstring here more verbose and clear (and then similarly update the other files too)?